# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/dietlibc/pkg_patch/pkg_e2fsprogs.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./misc/tune2fs.c.orig	Tue May  7 01:12:40 2002
+++ ./misc/tune2fs.c	Tue May  7 01:13:10 2002
@@ -406,23 +406,6 @@
 		print_label++;
 }
 
-static time_t parse_time(char *str)
-{
-	struct	tm	ts;
-
-	if (strcmp(str, "now") == 0) {
-		return (time(0));
-	}
-	memset(&ts, 0, sizeof(ts));
-	strptime(optarg, "%Y%m%d%H%M%S", &ts);
-	if (ts.tm_mday == 0) {
-		com_err(program_name, 0,
-			_("Couldn't parse date/time specifier: %s"),
-			str);
-		usage();
-	}
-	return (mktime(&ts));
-}
 
 static void parse_tune2fs_options(int argc, char **argv)
 {
@@ -586,11 +569,6 @@
 				break;
 			case 's':
 				s_flag = atoi(optarg);
-				open_flag = EXT2_FLAG_RW;
-				break;
-			case 'T':
-				T_flag = 1;
-				last_check_time = parse_time(optarg);
 				open_flag = EXT2_FLAG_RW;
 				break;
 			case 'u':
